docs-site/docs-styles/docs-navbar.css

```diff
@@ -67,8 +67,32 @@ footer {
   cursor: pointer;
 }
 
+.remove{
+  display: inline-block;
+  margin-left:2px;
+}
+
 @media screen and (min-width: 1024px) {
   .btn-hamburger {
     display: none;
   }
 }
+
+@media screen and (max-width: 500px) {
+  #removed {
+    display: none;
+  }
+}
+
```

Comment:
Please remove this. It is not getting applied anywhere.

Reply:
@Maianki removed